tests/t_libtest.cpp

```diff
@@ -5,7 +5,7 @@
 
 using namespace regexbench;
 
-static void report(const std::map<std::string, size_t>& m)
+void report(const std::map<std::string, size_t>& m, void* p)
```

Comment:
1. Okay, I will.
2. When I use it as a library, I need other data in the callback function. So I added it.